src/pudl/scripts/dbt_helper.py

```diff
@@ -331,8 +331,8 @@ def update_row_counts(
 
     if not has_test and not has_existing_row_counts:
         return UpdateResult(
-            success=False,
-            message=f"No row count test defined for {table_name}, nothing to update.",
+            success=True,
+            message=f"No row count test defined for {table_name}, and no row counts found, nothing to update.",
```

Comment:
The case of "there are neither row counts nor tests" seems like a success to me -- the row counts and the table data tests are consistent with each other.  This happens when we run use `all` rather than enumerating specific tables.